### PR TITLE
Add documentation for number field for plugins

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Given that `number` as a setting type is supported and it's pretty useful, it should also be documented.

#### Ticket Link
https://community-release.mattermost.com/core/pl/kjwpuu5hffyhmb7amgsnsd71xr

### CODE REVIEW

File: model/manifest.go [hunk 1]
```diff
@@ -108,6 +111,7 @@ type PluginSettingsSchema struct {
 //      "description": "This is my plugin",
 //      "homepage_url": "https://example.com",
 //      "support_url": "https://example.com/support",
+//		"release_notes_url": "https://example.com/releases/v0.0.1",
```
Comment (hanzei) [inline model/manifest.go]: I forgot to add this in https://github.com/mattermost/mattermost-server/pull/13677

Comment (lieut-data) [reply model/manifest.go]: Indentation looks off here?

Comment (hanzei) [reply model/manifest.go]: Good catch :+1: Fixed

**Comment by marianunez** [comment]
Thoughts on adding `color` and `banner` as well?

**Comment by hanzei** [comment]
> Thoughts on adding `color` and `banner` as well?

I general these are valuable types for plugins to use. But I would like to keep add them in a separate PR. Each types requires proper testing which I only did for `number`.

**Pull request merged**